src/server/info_collector.h

```diff
@@ -87,6 +90,9 @@ class info_collector
         ::dsn::perf_counter_wrapper check_and_set_qps;
         ::dsn::perf_counter_wrapper check_and_mutate_qps;
         ::dsn::perf_counter_wrapper scan_qps;
+        ::dsn::perf_counter_wrapper duplicate_qps;
```

Comment:
Rename to `dup_qps`? so the three metrics have the same pefix to identify.

Reply:
It's not trivial because on replica side the perf-counter called 'replica*app.pegasus*duplicate_qps@<pid>'. So on the collector side, it must follow the same name.